CONTRIBUTING.md

```diff
@@ -20,7 +20,7 @@ Avoid exception handling. Because Hyrise is not a product, we do not have to rec
 - Use [u]int(8|16|32|64)_t instead of `int, long, uint` etc.
 - Include in this order: header for implementation file, c system, c++ system, other
 - If your templated methods/classes are used outside of your file, they have to be in the header. But if you only use them internally, you should place them in the cpp file.
-- Try to keep the size of your templated methods as small as possible (Thin Template idiom). If only a small part of your method depends on the template parameter, consider moving the rest into a non-templated method. This reduces compile times.
+- Try to keep the size of your templated methods as small as possible (thin template idiom). If only a small part of your method depends on the template parameter, consider moving the rest into a non-templated method. This reduces compile times.
```

Comment:
It's important enough to change it back but I think names of approaches or idioms could be treated as names and use upper case.